lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,13 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    punctuation = '!@#$%^&*()-_+=[{]};:\'"<>/?,.\\|/'
+    text_no_punctuation = ''.join(char for char in text if not char not in punctuation)
+    text = text.lower()
+    tokens = [char for char in text_no_punctuation if char.isalpha()]
```

Comment:
Can you explain, why it is important to use both `isalpha` and presence in `punctuation` checks?

Also, since you use comprehension, why not directly create a list via list comprehension.

Reply:
It is important to use both **isalpha** and **punctuatuion** because the first one is responsible for removing numbers from the text and the second one is responsible for removing different symbols

Reply:
Will `<punctuation mark>.isalpha()` return `True`?